src/components/AccessibilityGuide/DynamicContent.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { Meta } from "@storybook/addon-docs";
+
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta title={getCategory("Dynamic Content")} />
+
+# Dynamic Content
+
+## Table of Contents
+
+- [General Information](#general-information)
+- [ARIA Live Regions](#aria-live-regions)
+- [Managing Focus](#managing-focus)
+- [Anti-Patterns](#anti-patterns)
+- [NYPL Patterns](#nypl-patterns)
+- [Resources](#resources)
+
+## General Information
+
+Most websites have elements on that are updated dynamically. Typically, these
```

Comment:
The first sentence has a grammatical error in the middle.  Add `them` or remove `on`.  It could go either way, but I would remove `on`.